docs/history/changelog.md

```diff
@@ -36,6 +36,7 @@ Per the [normative and non-normative content and changes policy](https://docs.go
 ### Codelists
 
 * [#1201](https://github.com/open-contracting/standard/pull/1201) Add 'terminatedEarly' and 'terminatedSuccessfully' codes to the `contractStatus` codelist, to distinguish between successful completion and early termination of the contract.
+* [#1209](https://github.com/open-contracting/standard/pull/1209) Replace "electronic goods" with "digital goods" in the description of `goods` from the `procurementCategory` codelist, to prevent the confusion with electronic devices such as laptops.
```

Comment:
```suggestion
* [#1209](https://github.com/open-contracting/standard/pull/1209) Replace "electronic goods" with "digital goods" in the description of 'goods' from the `procurementCategory` codelist, to align with the description in the World Trade Organization's Agreement on Government Procurement, and to avoid confusion between electronic goods like computers and digital goods like software.
```